### ISSUE TITLE
Sort options

### ISSUE THREAD

**Issue by philipzae** [tchx84/Portfolio#50]
It would be good to be able to set the folder sorting through the hamburger menu. Options to be able to sort by the file name, date, type/extension and size and then independently in ascending or descending order.

**Comment by tchx84** [comment]
> It would be good to be able to set the folder sorting through the hamburger menu. Options to be able to sort by the file name, date, type/extension and size and then independently in ascending or descending order.

Similarly to what I replied in #51, Portfolio is not meant to do _all_ the things you would "expect" to see in other file managers.  Its goal is to provide that minimal set of features that will allow people to manage their files. There are more reasons to this, pleas take a look at this post https://blogs.gnome.org/tchx84/2020/12/21/portfolio-manage-files-in-your-phone/

That paragraph above is for you to understand what I am trying to do with Portfolio. Now, for this particular request, I could consider some minimal sorting options, e.g. like I did with the filters option in the menu.

Let me think a bit more about this.
